packages/application-shell/src/components/application-shell-provider/application-shell-provider.tsx

```diff
@@ -24,7 +24,7 @@ type TApplicationShellProviderProps = {
   apolloClient?: ApolloClient<NormalizedCacheObject>;
   environment: TApplicationContext<{}>['environment'];
   applicationMessages: TAsyncLocaleDataProps['applicationMessages'];
-  children: (args: { isAuthenticated: boolean }) => JSX.Element;
+  children: (args: { isAuthenticated: boolean }) => React.JSX.Element;
```

Comment:
Since we're already importing react in this file, I think it would be more convenient to import the `JSX` namespace instead of relying on the global React object.